util/usage/indicator_test.go

```diff
@@ -0,0 +1,84 @@
+package usage
+
+import (
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	wfv1 "github.com/argoproj/argo/pkg/apis/workflow/v1alpha1"
+)
+
+func TestIndicatorForPod(t *testing.T) {
+	now := time.Now()
+	zero := now.Add(-time.Since(now))
```

Comment:
Why isn't `zero` `now` here? I.e. why do we need a zero as opposed to just using now?